frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -1,6 +1,4 @@
 import React from "react";
-import { color } from "metabase/lib/colors";
-import { colors } from "metabase/lib/colors/palette";
```

Comment:
Unused imports. We have a problem with our TypeScript and TS/ESlint setup that it won't lint unused variables in TypeScript files. Not the end of the world, but annoying every time I saw this happens.